ocw-cli/cli_app.py

```diff
@@ -1232,11 +1180,11 @@ def settings_screen(header):
          if option == '4':
               screen.addstr(25, x/2, "Enter Spatial Gridding Option [Model, Observation or User]:")
               new_spatial_grid_option = screen.getstr()
-              if new_spatial_grid_option.lower() == 'model':
-                   spatial_grid_option = 'Model'
-                   spatial_grid_setting = model_lat_res
-                   note = "Spatial gridding option has changed successfully to {0}".format(spatial_grid_option)
-              elif new_spatial_grid_option.lower() == 'observation':
+              #if new_spatial_grid_option.lower() == 'model':
```

Comment:
@MJJoyce 
Regarding the first one: Actually this is the main reason for removing the option from CLI. Sorry for the confusion. We used to provide the regridding on model grid points is because we have  used only regridded model datasets. So whichever model we choose, their resolution and grid points are always the same. When climate scientists evaluating multiple models with different resolution, they do not regrid observation data and the other model data into grid points of one model. It is always regridding into observation data's grid points or into user specified grid points.  

Regarding the second one:
This is an issue of our regridding scheme rather than a reason for changing CLI code. Once we address prioritized issues, I will open a ticket on this problem.